openwisp_radius/admin.py

```diff
@@ -469,6 +482,11 @@ def change_view(self, request, object_id, form_url='', extra_context=None):
             extra_context=extra_context,
         )
 
+    def add_view(self, request, form_url='', extra_context=None):
+        extra_context = extra_context or {}
+        extra_context['help_text'] = self.help_text
+        return super().add_view(request, form_url, extra_context)
```

Comment:
This is going into the template for add_view but it is not rendering:

![image](https://user-images.githubusercontent.com/56037184/161338540-b09301ac-62b0-45f8-9b61-9e76dff9df87.png)

![image](https://user-images.githubusercontent.com/56037184/161338751-4ff9b1b3-5192-4183-aea5-416fcb2fdebf.png)